app/tests/test_healthchecks.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+from unittest.mock import patch
+from app.main import HealthCheck
+from app.sdxftp import SDXFTP
+from ftplib import Error
+
+
+class TestHealthCheck(unittest.TestCase):
+
+    @unittest.skip("This test needs a locally running rabbit mq")
```

Comment:
This needs to be a `@unittest.skipIf` decorator with a conditional that checks for the availability of the rmq cluster.